### ISSUE TITLE
API Level 0.45 may be enforced soon. Is botting still safe?

### ISSUE THREAD

**Issue by VBQL** [PokemonGoF/PokemonGo-Bot#5801]
So to all of you guys know, PokemonGoF is using the 0.43 API for communicating with the Niantic servers. However, not mentioning names but people on other botting communities has found out API level 0.45 is starting to be enforced. FPM (Fast Pokemon Map) is currently down, and may be down forever. Would it be safe to continue botting again since I can see another ban wave coming?

**Comment by Jcolomar** [comment]
At this time, the .45 API has been forced, currently we would recommend stopping your may be easily flagged

**Comment by VBQL** [comment]
Yep, Niantic played us again. Well done.

**Comment by MerlionRock** [comment]
Guys, this time the RE team work fast. They have being RE 0.45 since it came out and just received a successful ping from 0.45

The API will be updated to 0.45, please wait and do not bot for now.

**Comment by Gobberwart** [comment]
Let's be honest... botting is really no more or less safe than it's ever been. If niantic really wants to crack down on punishing botters, it wouldn't be THAT hard for them to identify bot behaviour, regardless of api version.

Nobody will ever try to *guarantee* that any release of a bot is safe from banning.

If you can't accept that risk, don't bot.

**Comment by reigelgallarde** [comment]
a bot is a bot... and is always never safe.. haha

**Comment by MerlionRock** [comment]
Ya, anyway, for your information, new API (0.45) is REed by Pokeframer and it is them who shared the hashing. So....

I know it sound weird, remember to credit them when using 0.45 API, and hopefully they will share again when the next update happens again....

**Comment by Cptspal** [comment]
Do you have any link to get this 0.45 API ?
Thx

**Comment by MZorzy** [comment]
https://www.reddit.com/r/pokemongodev/comments/5c76pv/045_hash_function_now_works_again_with_unicorn/

**Comment by MerlionRock** [comment]
@Cptspal not yet.

Wait for PogoDev RE to finish reversing the hash routine and rebuilding the algorithm.

**Comment by dogeatdog** [comment]
I noticed you guys switched to another repo for the pgoapi ? Any rationale behind this ?

```
  git clone in ./src/pgoapi exists with URL https://github.com/pogodevorg/pgoapi.git/
  The plan is to install the git repository https://github.com/sebastienvercammen/pgoapi.git
```

**Comment by MZorzy** [comment]
cause some .so or .dll missing files. but also on 0.43 api

**Comment by Leruu76** [comment]
Yup i just got hit with an update my bot is down. Hmmmm imma go pg mode and check if im banned. Lol

**Comment by ElliotRobot1** [comment]
@Leruu76 
https://github.com/PokemonGoF/PokemonGo-Bot/issues/5809

**Comment by VBQL** [comment]
Ok thanks
